training_controller/prepare_training_logs.py

```diff
@@ -24,47 +21,26 @@
 class PrepareTrainingLogs:
     def __init__(self):
         self.WORKING_DIR = os.getenv("TRAINING_DATA_PATH", "/var/opni-data")
-        self.ES_DUMP_DIR = os.path.join(self.WORKING_DIR, "windows")
+        self.TRAINING_DIR = os.path.join(self.WORKING_DIR, "windows")
+        self.ES_DUMP_DIR = os.path.join(self.WORKING_DIR, "esdump_path")
         self.ES_DUMP_SAMPLE_LOGS_PATH = os.path.join(
             self.WORKING_DIR, "sample_logs.json"
         )
-        self.es_dump_data_path = ""
-
-    def save_window(self, window_start_time_ns, df):
-        current_window_json_files = [
-            file
-            for file in os.listdir(self.ES_DUMP_DIR)
-            if str(window_start_time_ns) in file
-        ]
-        df[
-            [
-                "timestamp",
-                "window_start_time_ns",
-                "masked_log",
-                "is_control_plane_log",
-            ]
-        ].to_json(
-            os.path.join(
-                self.ES_DUMP_DIR,
-                "{}_{}.json.gz".format(
-                    window_start_time_ns, len(current_window_json_files)
-                ),
-            ),
-            orient="records",
-            lines=True,
-            compression="gzip",
-        )
-
+    '''
+    Fetch size of disk
+    '''
     def disk_size(self):
-        # Fetch size of disk
         logging.info("Fetching size of the disk")
         total, used, free = shutil.disk_usage("/")
         logging.info("Disk Total: %d GiB" % (total // (2 ** 30)))
         logging.info("Disk Used: %d GiB" % (used // (2 ** 30)))
         logging.info("Disk Free: %d GiB" % (free // (2 ** 30)))
         return free
-
+    '''
+    Fetch the logs from each time interval within query_commands using Elasticdump.
+    '''
```

Comment:
Comments should always be under a function
i.e put under `def run_esdump(self, query_commands):`

Reply:
This applies to all other multiline comments in this file